editor/src/clj/editor/pipeline/bob.clj

```diff
@@ -360,9 +360,10 @@
       {:code 302
        :headers {"Location" (str html5-url-prefix "/index.html")}}
 
-      (let [served-file   (try-resolve-html5-file project url)
+      (let [url-without-query-params  (get (string/split url #"\?") 0)
```

Comment:
It will also fail for URL fragments, right? E.g. `"/foo#some-anchor"`... We could add checking for `#` to the split regex, or we could parse URL and extract the path (assuming `url` here is of the form `/foo/bar?q=1#zoops`):
```clj
(.getPath (java.net.URL. (str "http://" url)))
=> "/foo/bar"

```

Reply:
Ah, yes, you're right. It might be better to parse it.